### PR TITLE
Updated info for Windows developers

### PR DESCRIPTION
**Pull request by wwwizzarrdry**
Added a not-so-obvious tip for addressing the issue of animations appearing to be broken on Windows machines.
TL;DR 
Change **Visual Effects** to **Adjust for Best Appearance** to enable animations.

**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [x] It's been tested on an Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs)  or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to [start a new feature discussion](https://github.com/quasarframework/quasar/discussions/new?category=ideas-proposals) first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by wwwizzarrdry** [comment]
I was losing my mind trying to resolve this in code. Last thing I ever thought to check was an OS level setting.

**Comment by rstoenescu** [comment]
Thanks for contributing!
